<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicSet</class>
 <widget class="QDialog" name="BasicSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>538</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 248, 255);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>149</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/cs351set.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>Controller设置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>87</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>CH1 IP：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_controllerIp_ch1">
     <property name="geometry">
      <rect>
       <x>279</x>
       <y>90</y>
       <width>181</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_controllerIp_ch2">
     <property name="geometry">
      <rect>
       <x>279</x>
       <y>153</y>
       <width>181</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>CH2 IP：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>262</y>
       <width>80</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
background-color: rgb(51, 153, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>149</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
background-color: rgb(51, 153, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>64</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
font: 14pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;
}
QComboBox::drop-down {
width:30px; 
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
}
QComboBox::down-arrow {
image: url(:/LCD_CS351/LCD_CS351/logo/arrow.png);
}
QComboBox  QAbstractItemView{
height:80px;
border: 1px solid rgb(51, 153, 255); 
selection-background-color: rgb(51, 153, 255); 
}
QScrollBar:vertical{
width:30px;
}
QComboBox QAbstractItemView::item{
height:20px;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>123</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>CH2：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>CH1：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
font: 14pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;
}
QComboBox::drop-down {
width:30px; 
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
}
QComboBox::down-arrow {
image: url(:/LCD_CS351/LCD_CS351/logo/arrow.png);
}
QComboBox QAbstractItemView{
height:50px;
border: 1px solid rgb(51, 153, 255); 
selection-background-color: rgb(51, 153, 255); 
}
QScrollBar:vertical{
width:30px;
}</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>200</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
font: 14pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;
}
QComboBox::drop-down {
width:30px; 
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
}
QComboBox::down-arrow {
image: url(:/LCD_CS351/LCD_CS351/logo/arrow.png);
}
QComboBox QAbstractItemView{
height:50px;
border: 1px solid rgb(51, 153, 255); 
selection-background-color: rgb(51, 153, 255); 
}
QScrollBar:vertical{
width:30px;
}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>193</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>工厂：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style: solid;
background-color: rgb(51, 153, 255);</string>
     </property>
     <property name="text">
      <string>清空数据库</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>275</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>149</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/serve set.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>166</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>曲线服务器：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_DataServerIp">
     <property name="geometry">
      <rect>
       <x>301</x>
       <y>111</y>
       <width>161</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>108</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>数据服务器：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>服务器设置</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CurveServerIp">
     <property name="geometry">
      <rect>
       <x>301</x>
       <y>170</y>
       <width>161</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_AndonServerIp">
     <property name="geometry">
      <rect>
       <x>301</x>
       <y>230</y>
       <width>161</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>226</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>Andon服务器：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>290</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>端口：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_port">
     <property name="geometry">
      <rect>
       <x>301</x>
       <y>294</y>
       <width>161</width>
       <height>34</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>149</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
background-color: rgb(51,153,255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_BarcodeGun">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>196</y>
       <width>161</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>条码来源设置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>149</x>
       <y>193</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>条码枪波特率：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEdit_BarcodeGun</cstring>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>253</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QRadioButton::indicator { 
   width: 25px;
   height: 25px;
   image: url(:/LCD_CS351/LCD_CS351/logo/Radio.png);
}

QRadioButton::indicator:checked { 
image: url(:/LCD_CS351/LCD_CS351/logo/Radio_check.png);
}</string>
        </property>
        <property name="text">
         <string>有线</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QRadioButton::indicator { 
   width: 25px;
   height: 25px;
   image: url(:/LCD_CS351/LCD_CS351/logo/Radio.png);
}

QRadioButton::indicator:checked { 
image: url(:/LCD_CS351/LCD_CS351/logo/Radio_check.png);
}</string>
        </property>
        <property name="text">
         <string>无线</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="lineEdit_RfidIp">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>181</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_RfidPort">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>143</y>
       <width>111</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>140</y>
       <width>120</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>RFID端口号：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEdit_RfidPort</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>87</y>
       <width>120</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>RFID IP：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>lineEdit_RfidIp</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>246</y>
       <width>120</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>条码枪模式：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_142">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/logo/Barcode.png);
background:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_37">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>38</width>
     <height>38</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/35_all/cha.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_38">
   <property name="geometry">
    <rect>
     <x>482</x>
     <y>295</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/LCD_CS351/LCD_CS351/35_all/67.bmp);
font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>pushButton_38</zorder>
  <zorder>pushButton_37</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
